lib/ExpensiMark.js

```diff
@@ -367,7 +370,9 @@ export default class ExpensiMark {
             if (abort) {
                 replacedText = replacedText.concat(textToCheck.substr(match.index, (match[0].length)));
             } else {
-                replacedText = replacedText.concat(replacement(match[0], match[1], match[2], match[3]));
+                // we are allowing only bold, strikethrough and italic formatting for link text
```

Comment:
Comment can be improved. Mention why are we doing this. Comment starts from capitals.